src/docs/manual/03_task_exportDrawIo.adoc

```diff
@@ -1,15 +1,45 @@
 :filename: manual/03_task_exportDrawIo.adoc
 ifndef::imagesdir[:imagesdir: ../images]
 
+
 = exportDrawIo
 
 include::feedback.adoc[]
 
-There is no `exportDrawIo` task because it is not needed.
-When you want to use https://draw.io[draw.io] for your diagrams,
-use the free offline version and save your diagrams with the `.png` file extension
-(or even better `.dio.png` to easily recognize draw.io images).
+There is no `exportDrawIo` task because it is not required. You can continue to use https://draw.io[draw.io] as an editor for your diagrams by making a change to your diagram authoring workflow.
+
+Export your draw.io diagrams as a PNG with the source embedded in the file metadata.
+Using this approach allows you to embed your diagrams into AsciiDoc source as you normally do with the `image::` macro, with the added advantage of storing the diagram source with the image itself.
+
+NOTE: If you are converting a Confluence page with embedded draw.io diagrams to AsciiDoc, you can use this export and editing workflow to continue to enjoy the draw.io editing experience. 
+
+.Export an editable PNG diagram from Confluence 
+. Load the diagram you want to export from Confluence.
+. Click menu:File[Export as > PNG...]
+. In the Image modal, make sure that "Include a copy of my diagram" is selected.
+. Click Export to save the PNG file with the pattern `[file].dio.png`.
++
+NOTE: Specifying `.dio` (short for "**d**raw**io**") in the name will help you identify PNG files containing embedded XML diagram source.
++
+. Commit the exported PNG file to source control.
+
+You now have a diagram that can be managed in source control and edited using draw.io. 
+
+.Edit a diagram in Draw.io Offline editor mode
```

Comment:
Do we need this section? I think it should be replaced with a hint to the Desktop version: https://about.draw.io/integrations/#integrations_offline

With this version, you can edit your diagrams offline and don't have to take care of duplicate files.

Reply:
This is a much better idea. I just installed the Windows desktop version and its heaps better than the offline web editor. I was just going with what I thought was suggested in the original text.